### DISCUSSION TITLE
WebRadios - Logo folders

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#1035] [Fixed Issues]
I know the folders used for the WebRadios logs changed recently and I made the changes noted in the release notes.

I have just been adding a few new radio stations which went OK until I tried to update the relevant logos. Of the three new stations I was only able to add one logo.  When I checked the folder structures they weren't as I remembered but I can't find anything in the wiki pages to confirm what they should be.  I use SharedData and right now have:

/srv/http/data
                       /webradio                  <-- contains the webradio station config files
                                       /img           <-- contains the webradio staion logos

Is this currently correct ?

Thanks

Chris

**Comment by ceejayemm** [comment]
I now don't think its a problem with the Logos folder but with how rAudio-1 handles the station url.  The two stations in question are:

Easy UK (was Breezy Radio UK): http://listen-nation.sharp-stream.com/breezyradiouk.mp3?ref=RF

Magic UK: http://edge-bauermz-01-gos2.sharp-stream.com/magicnational.mp3?aw_0_1st.skey=1663924539

For Easy UK, if I remove '?ref=RF' from the end of the URL then the logo will display and the station plays.  However doing the same for Magic UK (removing '?aw_0_1st.skey=1663924539' allows the logo to display BUT the station will NOT play.

Any solution to get these stations to display their logo AND to play ?

Thanks

Chris

**Reply by rern** [reply]
Hi Chris,

Today's UPDATE branch should resolve this issue.

**Reply by ceejayemm** [reply]
@rern 

Have applied latest Update branch, still the same problem.

Chris

**Comment by ceejayemm** [comment]
@rern 

I updated to 20221010 today just to get myself back on to the main branch. This problem with Magic UK still seems to be a problem but I think I now know what the cause is.  The problem relates to the '?aw_0_1st.skey=' parameter at the end of the station URL.  This seems to change on a regular (daily ?) basis. For example, when I first reported this problem the Magic UK URL was:
http://edge-bauermz-01-gos2.sharp-stream.com/magicnational.mp3?aw_0_1st.skey=1663924539

Yesterday it was:
http://edge-bauermz-01-gos2.sharp-stream.com/magicnational.mp3?aw_0_1st.skey=1665422123

and today it is:
http://edge-bauermz-01-gos2.sharp-stream.com/magicnational.mp3?aw_0_1st.skey=1665475077

Is there any way rAudio-1 can cope with this regular change to the URL for a web radio station ?  With the correct daily (?) URL rAudio-1 will play the Magic UK station  but obviously will not play once the URL changes.

Chris
